src/options/options.js

```diff
@@ -6,10 +6,16 @@
 
 import * as AddonSettings from "/common/modules/AddonSettings/AddonSettings.js";
 import * as AutomaticSettings from "/common/modules/AutomaticSettings/AutomaticSettings.js";
+import * as RandomTips from "/common/modules/RandomTips/RandomTips.js";
 
 import * as CustomOptionTriggers from "./modules/CustomOptionTriggers.js";
 
 // init modules
 CustomOptionTriggers.registerTrigger();
 AutomaticSettings.setDefaultOptionProvider(AddonSettings.getDefaultValue);
 AutomaticSettings.init();
+
+RandomTips.init("/common/modules/data/Tips.js").then(() => {
```

Comment:
I now get a different error: 
![image](https://user-images.githubusercontent.com/15305150/118131529-7cd84c00-b3b3-11eb-86c8-53d1e0d02009.png)

I think that this line needs to be updated now that you have reverted the changes to RandomTips…

Reply:
uups well… of course… fixed.